_people/amy_naylor.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Amy Naylor
+gender: female
+submitted: 2018-01-28
+contact_allowed: true
+graduated: 2017
+course:
+ - English
+
+---
+
+MPhil Critical Approaches to Children's Literature, University of Cambridge
```

Comment:
If it's going in as a bio potentially add some filler rather than just state the course? Makes it read better as a Bio